django_project/api/fixtures/hxl_tags.json

```diff
@@ -0,0 +1,29 @@
+{
+  "uuid": "#meta +id",
+  "lat": "#geo+lat",
+  "lon": "#geo+lon",
+  "amenity": "#loc+amenity",
+  "healthcare": "#meta+healthcare",
+  "name": "#loc +name",
+  "operator": "#meta +operator",
+  "source": "#geo+bounds+url",
+  "speciality": "#meta +speciality",
+  "operator_type": "#meta +operator_type",
+  "addr_full": "#contact +address",
+  "contact_number": "#contact +phone",
+  "operational_status": "#status+operational_status",
+  "opening_hours": "#access +hours",
+  "beds": "#capacity +beds",
+  "staff_doctors": "#capacity +staff",
+  "staff_nurses": "#capacity +staff",
+  "health_amenity_type": "#meta +health_amenity_type",
+  "dispensing": "#meta+dispensing",
+  "wheelchair": "#meta+wheelchair",
+  "emergency": "#meta+emergency",
+  "insurance": "#meta+insurance",
+  "water_source": "#meta+water_source",
+  "electricity": "#meta+electricity",
+  "is_in_health_area": "#meta+is_in_health_area",
+  "is_in_health_zone": "#meta+is_in_health_zone",
+  "url": "#contact +url"
+}
```

Comment:
Do #hxl tags have spaces in them or not - it seems pretty inconsistent? @markherringer can you check?